apps/myaccount/src/components/profile/fields/checkbox-field-form.tsx

```diff
@@ -0,0 +1,127 @@
+/**
+ * Copyright (c) 2025, WSO2 LLC. (https://www.wso2.com).
+ *
+ * WSO2 LLC. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import { Field, FormValue, Forms } from "@wso2is/forms";
+import isEmpty from "lodash-es/isEmpty";
+import React, { FunctionComponent, ReactElement } from "react";
+import { Grid, List } from "semantic-ui-react";
+import { CheckBoxFieldFormPropsInterface } from "../../../models/profile-ui";
+import { EditSection } from "../../shared/edit-section";
+
+const CheckboxFieldForm: FunctionComponent<CheckBoxFieldFormPropsInterface> = ({
+    fieldSchema: schema,
+    initialValue,
+    fieldLabel,
+    setIsProfileUpdating,
+    handleSubmit,
+    isUpdating,
+    ["data-componentid"]: testId
```

Comment:
```suggestion
    ["data-componentid"]: componentId
```

Reply:
Fixed via dcc6825ac27442053d276327da8ed8f5c8ab398e

Other places will be updated via a follow up PR.